web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml

```diff
@@ -16,21 +16,19 @@ permissions:
   - 'request collection archival'
   - 'update group'
   - 'view any unpublished content'
-  - 'view any unpublished news content'
   - 'create custom_page content'
   - 'edit any custom_page content'
   - 'delete any custom_page content'
   - 'edit any document content'
   - 'delete any document content'
   - 'edit any event content'
   - 'delete any event content'
-  - 'create rdf entity news'
+  - 'create rdf entity content'
+  - 'view any unpublished news content'
   - 'create news content'
   - 'edit any news content'
   - 'delete any news content'
-  - 'update any navigation ogmenu_instance'
-  - 'add new links to og menu instance entities'
-  - 'create asset_distribution rdf_entity'
-  - 'create asset_release rdf_entity'
-  - 'create solution rdf_entity'
+  - 'create asset_distribution rdf entity'
+  - 'create asset_release rdf entity'
+  - 'propose solution rdf entity'
```

Comment:
`create solution rdf entity` is changed into `propose solution rdf entity`, but there is no code or tests in this PR that refer to this change. It is also not mentioned in the ticket that we need to change anything about solutions.